mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,52 +1,72 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$iconSize: 32px;
+$privacyLineHeight: 140%;
+$socialTextFontWeight: 500;
+$rulerHeight: 1px;
 
 .footer {
   background-color: var(--quaternaryBgColor);
 }
 
 .footerWrapper {
-  width: 100%;
   max-width: $maxWidthContainer;
-  justify-content: space-between;
-  padding: $paddingBig $paddingMedium;
-  border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
+  box-sizing: content-box;
```

Comment:
let's don't use content-box. It's not consisnently that in all project we use border-box and here content box

Reply:
Ok, I removed it. But now I need to use calc() function for max-width to have the same behavior. As for me, the code was easier to read with "box-sizing: content-box;"